rotkehlchen/tests/unit/decoders/test_convex.py

```diff
@@ -39,49 +41,11 @@ def test_convex_pools(ethereum_inquirer):
     """Tests that our hardcoded information about convex pools is up-to-date.
     Queries data about convex pools reward addresses and their names from chain and compares it
     to the current hardcoded info."""
-    booster_contract = ethereum_inquirer.contracts.contract(string_to_evm_address('0xF403C135812408BFbE8713b5A23a04b3D48AAE31'))  # noqa: E501
-    pools_count = booster_contract.call(
-        node_inquirer=ethereum_inquirer,
-        method_name='poolLength',
-    )
-    calls_to_booster = [(
-        booster_contract.address,
-        booster_contract.encode('poolInfo', [x]),
-    ) for x in range(pools_count)]
-    booster_result = ethereum_inquirer.multicall(
-        calls=calls_to_booster,
-    )
-    convex_rewards_addrs = []
-    convex_lp_tokens_addrs = []
-    lp_tokens_contract = EvmContract(  # only need it to encode and decode
-        address=ZERO_ADDRESS,
-        abi=ethereum_inquirer.contracts.abi('CONVEX_LP_TOKEN'),
-        deployed_block=0,
-    )
-    for single_booster_result in booster_result:
-        lp_token_addr = hex_or_bytes_to_address(single_booster_result[0:32])
-        crv_rewards = hex_or_bytes_to_address(single_booster_result[3 * 32:4 * 32])
-        convex_rewards_addrs.append(crv_rewards)
-        convex_lp_tokens_addrs.append(lp_token_addr)
-
-    # We query this info from chain instead of using data from our assets database since
-    # if convex adds a new pool with new lp token we won't know its properties (because it won't
-    # be in our DB)
-    calls_to_lp_tokens = [(lp_token_addr, lp_tokens_contract.encode('symbol')) for lp_token_addr in convex_lp_tokens_addrs]  # noqa: E501
-    lp_tokens_result = ethereum_inquirer.multicall(
-        calls=calls_to_lp_tokens,
-    )
-
-    queried_convex_pools_info = {}
-    for convex_reward_addr, single_lp_token_result in zip(convex_rewards_addrs, lp_tokens_result):
-        decoded_lp_token_result = lp_tokens_contract.decode(single_lp_token_result, 'symbol')
-        queried_convex_pools_info[convex_reward_addr] = decoded_lp_token_result[0]
-
-    if queried_convex_pools_info != CONVEX_POOLS:
-        added_pools_addrs = queried_convex_pools_info.keys() - CONVEX_POOLS.keys()
+    if (queried_convex_pools_info := query_convex_data_from_chain(ethereum_inquirer)) != (convex_pools := read_convex_data_from_cache()):  # noqa: E501
+        added_pools_addrs = queried_convex_pools_info.keys() - convex_pools.keys()
```

Comment:
we have a test that is VCR for this kind of checks for curve in `def test_curve_cache(rotkehlchen_instance, use_curve_api):`. Maybe you can move this test and follow a similar approach since it doesn't make much sense anymore for it to be here